pages/apis/graphql/cookbooks/builds.md

```diff
@@ -288,3 +288,19 @@ Then, create the build:
     }
   }
 ```
+## Get the webhook payload of a build
+
+This query allows you to fetch the webhook payload of a specific build using its UUID. The payload is only available for 7 days
```

Comment:
Thanks for your PR @stephanieatte !

Just adding a full-stop/period to the end of the last sentence 👍 

```suggestion
This query allows you to fetch the webhook payload of a specific build using its UUID. The payload is only available for 7 days.
```

Reply:
Added the . @gilesgas